### PR TITLE
fix(security): update npm to 6.13.4 version

### PR DESCRIPTION
**Pull request by rap2hpoutre**
Previous version has been detected as a security issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,8 +72,9 @@
     "husky": "3.1.0",
     "jest": "24.9.0",
     "nock": "11.7.0",
+    "npm": "6.13.4",
```
Comment (larcin) [inline package.json]: [MISC] We hadn't npm at all in the first place ? If so, why do we need it now ?

Comment (rap2hpoutre) [reply package.json]:
```
forest-express git:(chore/message-length) yarn why npm
yarn why v1.19.1
[1/4] 🤔  Why do we have the module "npm"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "npm@6.13.2"
info Reasons this module exists
   - "semantic-release#@semantic-release#npm" depends on it
   - Hoisted from "semantic-release#@semantic-release#npm#npm"
info Disk size without dependencies: "37.21MB"
info Disk size with unique dependencies: "53.14MB"
info Disk size with transitive dependencies: "69.51MB"
info Number of shared dependencies: 214
✨  Done in 1.13s.
```

Comment (larcin) [reply package.json]: 🙏

**Comment by arnaudbesnier** [comment]
@rap2hpoutre Is it really necessary to add `npm` in the `devDependencies` to fix the vulnerabilities?
Shouldn't `semantic-release` reinstall do the job?
(according to this dependency rule: https://github.com/semantic-release/semantic-release/blob/master/package.json#L28)

**Comment by arnaudbesnier** [comment]
I confirm it does not fix the vulnerability:
- `yarn audit` still mentions npm vulnerability.
- and Github still mentions the vulnerability
Screenshot 2020-01-22 at 10 51 16


(Good read? https://www.rockyourcode.com/fix-npm-vulnerabilities-with-yarn/)

**Comment by arnaudbesnier** [comment]
I'll amend the commit on `devel` to fix this.

**Comment by arnaudbesnier** [comment]
Final commit here:
https://github.com/ForestAdmin/forest-express/commit/417bd1967ebcaa2034b465a3f942fff57ecbd17e

**Pull request merged**